packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/DbSyncNetworkInfoProvider.ts

```diff
@@ -15,36 +15,47 @@ import { NetworkInfoBuilder } from './NetworkInfoBuilder';
 import { NetworkInfoCacheKey } from '.';
 import { Pool } from 'pg';
 import { Shutdown } from '@cardano-sdk/util';
-import { loadGenesisData, toGenesisParams, toLedgerTip, toStake, toSupply, toWalletProtocolParams } from './mappers';
-import { pollDbSync } from './utils';
+import { epochPollService } from './utils';
+import {
+  loadGenesisData,
+  toGenesisParams,
+  toLedgerTip,
+  toStake,
+  toSupply,
+  toTimeSettings,
+  toWalletProtocolParams
+} from './mappers';
 
 export interface NetworkInfoProviderProps {
   cardanoNodeConfigPath: string;
-  dbPollInterval: number;
+  epochPollInterval: number;
 }
 export interface NetworkInfoProviderDependencies {
   db: Pool;
   cache: InMemoryCache;
   logger?: Logger;
+  cardanoNode: CardanoNode;
 }
 export class DbSyncNetworkInfoProvider extends DbSyncProvider implements NetworkInfoProvider {
```

Comment:
The name of `DbSyncNetworkInfoProvider` is a little misleading since it's now also using CardanoNode. Don't have a good suggestion though.